WordPress/Docs/WhiteSpace/CastStructureSpacingStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Cast Structure Spacing">
+    <standard>
+    <![CDATA[
+    Surround type casts with spaces.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: space after typecast.">
+        <![CDATA[
+$a = (int) '420';
+        ]]>
+        </code>
+        <code title="Invalid: space after typecast.">
+        <![CDATA[
+$a = (string)420;
```

Comment:
It will be easier to spot the difference if everything apart from the key part of the standard (spaces around casting) is left the same. I'd suggest changing this to:

```php
$a =(int)'420';
```

That's literally a paste of the valid example, with the spaces around the cast removed.

Reply:
Sorry, my bad. Adjusted it.